spec/features/user_edit_profile_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+RSpec.describe 'Users profile page', driver: :selenium_headless, type: :feature do
```

Comment:
This test file is quite long, maybe separate it into some `context`s (e.g. for different routes)?